std_msgs/Point32 center
std_msgs/Point32 extents
std_msgs/Point32 axis
float32 angle
